lib/dhl_uk/errors/error.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module DhlUk
+  module Errors
+    class Error < StandardError
+      attr_reader :response, :status, :reason
+
+      def initialize(response)
```

Comment:
Should `response` be converted to keyword param to keep it consistent with `ResponseError`? Or perhaps the other way around?

Reply:
The other way round. I will change.